### DISCUSSION TITLE
If use rifle's scope to zoom in in this map, shadows will disappear, what happed? Is this map is too large?

### DISCUSSION THREAD

**Discussion by ghost** [redeclipse/base#1279] [Help and Support] [answered]
[maps.zip](https://github.com/redeclipse/base/files/6785327/maps.zip)

**Comment by ghost** [comment]
shadow not shadows

**Comment by cloudziq** [comment, marked as answer]
Engine limitations, or buggy zoom code,  I have no idea but definitelly zoom works strange
you can semi-fix it by setting `/csmfarplane 4096` in editmode.  But this will reduce shadows quality
